src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const object = {};
+
+  const splittedSource = String(sourceString).split(';');
+
+  splittedSource.forEach((item) => {
+    const [key, value] = item.split(':');
```

Comment:
The current implementation assumes that each item will have both a key and a value. Consider adding error handling for cases where `item.split(':')` does not return exactly two elements, which could happen if the input string is malformed.